plugins/postcss-is-pseudo-class/src/split-selectors/compound-selector-order.ts

```diff
@@ -1,7 +1,7 @@
 import parser from 'postcss-selector-parser';
 
 export function sortCompoundSelectorsInsideComplexSelector(node) {
-	if (!node || !node.nodes) {
+	if (!node || !node.nodes || node.nodes.length === 1) {
```

Comment:
`:is(a)` cause issues and doesn't require sorting anyway.
Better to short circuit.